src/SourceGenerators/Uno.UI.SourceGenerators/XamlGenerator/XamlCodeGeneration.cs

```diff
@@ -229,6 +230,11 @@ public XamlCodeGeneration(GeneratorExecutionContext context)
 				_isHotReloadEnabled = _isDebug;
 			}
 
+			if (!bool.TryParse(context.GetMSBuildPropertyValue("UnoGenerateXamlSourcesProvider"), out _generateXamlSourcesProvider))
```

Comment:
Are we concerned that this could mean someone could accidentally include the source code in a release build by setting the UnoGenerateXamlSourceProvider to true?

Reply:
It's only to disable the generation if any issue arises for an unforeseen generation issue.